File: loianeTraining/javaBasico/exercicios-aulas-18-19/Ex32.java
class Ex32 {
    public static void main(String[] args) {
        int[] a = { 1, 2, 3, 4, 5 };

        for (int i = 0; i < a.length; i++) {
            for (int j = 0; j <= 10; j++) {
                System.out.println(a[i] + " x " + j + " = " + a[i] * j);
            }
            System.out.println();
        }
    }
}

File: loiane-training-csharp/procedural-poo/e/Ex32.cs
using System;

class Ex32
{
    static void Main(string[] args)
    {
        int[] vetorA = new int[5] { 1, 3, 5, 7, 9 };

        for (int i = 0; i < vetorA.Length; i++)
        {
            for (int j = 1; j <= 10; j++)
            {
                Console.WriteLine("{0} X {1} = {2}", j, vetorA[i], vetorA[i] * j);
            }
            Console.WriteLine();
        }
    }
}
